About Us

claynaut opened the pull request:
<!-- Please follow the provided template -->
### 1. Please briefly explain what it is you coded
Implemented About Us page based on Figma design.

### 2. Please briefly explain your approach (new components added? modified? removed?)
Use a combination of Material UI components to code the About Us page.

### 3. Any packages added/updated/removed
N/A

### (Optional) 4. Any feature recommendations/insights/questions/comments/etc.?

kadey001 committed:
chore(updated packages): updated codegen cli package

kadey001 committed:
chore(yarn): updated yarn to version 3 and cache updated

kadey001 committed:
test(locust): added a locust load testing script

kadey001 committed:
fix(typescript-ide): fixed typescript ide problems by running `yarn dlx @yarnpkg/sdks vscode`

kadey001 committed:
fix(questionCarousel): temporary fix for question carousel subscription updates

kadey001 committed:
fix(SettingsMenu): reverted to flat expanded settings menu

claynaut committed:
Merge branch 'c2c-project:dev' into dev

Commit:
Revert "feat(frontend): implement dashboard"

This reverts commit c88482c77584ba8ba56f7ae119ac499c8e1815ff.

Commit:
Merge branch 'dev' of https://github.com/claynaut/prytaneum into dev

kadey001 committed:
ci(deployment): removed unused fields in config file

kadey001 committed:
ci(github action): added template for gke deployment action

kadey001 committed:
Merge remote-tracking branch 'c2c/dev' into CI/CD

kadey001 committed:
fix(client build): fixed client build errors

claynaut committed:
Merge branch 'c2c-project:dev' into dev

kadey001 committed:
ci(dockerized+added github action): dockerized both the front and backend seperatly

kadey001 committed:
Update gke.yml

kadey001 committed:
Updated gke.yml secret var

kadey001 committed:
Updated cluster name

kadey001 committed:
Updated build path for dockerfiles

kadey001 committed:
Updated docker build commands

kadey001 committed:
Fixed server build step

kadey001 committed:
Server build fix

kadey001 committed:
ci(client dockerfile): updated dockerfile to reduce client container size

kadey001 committed:
Merge branch 'CI/CD' of https://github.com/kadey001/prytaneum into CI/CD

kadey001 committed:
Added kustomize commands for gke.yml

kadey001 committed:
fix(server env): updated server env to include db vars

kadey001 committed:
Merge branch 'CI/CD' of https://github.com/kadey001/prytaneum into CI/CD

kadey001 committed:
fix(client env): removed client env file

kadey001 committed:
fix(client env): added back client env w/ new url

kadey001 committed:
fix(cors): added resonse cors headers

kadey001 committed:
ci(kustomize): commented out customize for now

Commit:
feat(frontend): intitial commit for user guide

kadey001 committed:
fix(cookies): updated cookie options for cors

Commit:
feat(frontend): complete user guide

Commit:
feat(frontend): text size adjustment

kadey001 committed:
fix(kubernetes): added heath checking routes for kubernetes

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): implement about us page

Commit:
feat(frontend): enlarge paragraph text

kadey001 committed:
ci(updated deployment): gke deployments seperated

kadey001 committed:
Update gke.yml

Updated kustomize command

kadey001 committed:
Update gke.yml

kustomize build path

kadey001 committed:
ci(server deployment): updated yml

kadey001 committed:
ci(client service): fixed client service labels & name

kadey001 committed:
ci(namespace + config): updated namespace & server configmap

kadey001 committed:
ci(configmap): updated configmaps

kadey001 committed:
ci(config map fix): int -> str for port vars

kadey001 committed:
ci(ingress): updated ingress and traffic policy

kadey001 committed:
fix(ingress): updated config

kadey001 committed:
Merge pull request #137 from claynaut/user-guide

User Guide

kadey001 committed:
refactor(client): updated images from remote url to local static images

kadey001 committed:
ci(deployment): seperated out deployments

kadey001 committed:
ci(paths & order): updated cd paths and order of steps

kadey001 committed:
ci(rollout checks): seperated out checks

kadey001 committed:
ci(client rollout): commented out for testing

kadey001 committed:
ci(namespace): added namespace to rollout checks

kadey001 committed:
ci(removed old deployment): old deployment file deleted + renamed github action file

kadey001 committed:
ci(cleaned up deployment): minor cleanup

kadey001 committed:
Merge pull request #125 from kadey001/CI/CD

CI/CD Pipeline for deploying on GKE & seperated out front/backend containers.

kadey001 committed:
Merge remote-tracking branch 'c2c/dev' into dev

Commit:
feat(frontend): implement dashboard

kadey001 committed:
Merge branch 'dev' into CI/CD

kadey001 committed:
Merge remote-tracking branch 'c2c/dev' into CI/CD

kadey001 committed:
ci(formatted + dev deployment): namespace more explicit now

kadey001 committed:
Merge pull request #3 from kadey001/CI/CD

CI/CD Pipeline for Dev & Production + Split out frontend and backend

kadey001 committed:
ci(formatting): fixed yml formatting

kadey001 committed:
fix(dev deployment): fixed tag var

kadey001 committed:
Merge branch 'CI/CD' into dev

kadey001 committed:
fix(updated kustomize file): removed resource that was moved

kadey001 committed:
fix(kubernetes): updated db-secret file name

kadey001 committed:
ci(ingress): updated ingress file locations & added a dev version

kadey001 committed:
ci(ingress): updated ingress file locations & added a dev version

kadey001 committed:
Merge branch 'dev' of https://github.com/kadey001/prytaneum into dev

kadey001 committed:
ci(action name update): fixed dev version name

kadey001 committed:
ci(typo): fix ingress file name

kadey001 committed:
ci(dev): updated regional static ip name for dev deployment

kadey001 committed:
ci(prod fix): temp branch change for prod fix

kadey001 committed:
ci(typo+dev): updated ingress for dev

kadey001 committed:
Merge pull request #139 from claynaut/dashboard-redo

Dashboard

kadey001 committed:
ci(updated dev deployment): updated client for dev deployment

kadey001 committed:
ci(dev deployment): removed quotes from DEV_GRAPHQL secret

kadey001 committed:
ci(dev): updated ingress

kadey001 committed:
ci(dev): re-added static-ip name

kadey001 committed:
ci(dev): updated GRAPHQL_URL & fixed typo in dockerifle

kadey001 committed:
Merge pull request #141 from kadey001/dev

Set up Cloud Deployment pipeline

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
         width: '100%',
-        height: '100%',
+        minHeight: '85vh',
+        paddingLeft: '1rem',
```
Rather than rem, lets use `theme.spacing()` -- probably `theme.spacing(1)`

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
 import * as React from 'react';
```
Name of this file should be in PascalCase -- `AboutUs.tsx`

claynaut replied:
For some reason, when I try to access the page in PascalCase, it reverts to lowercase.

DevelopmentByDavid replied:
Ah, my mistake.  I didn't originally notice that this was in `pages/`.  I think it should then be `kebab-case` -- `about-us`.  That would change the URL too.

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
     },
-    paper: {
-        padding: theme.spacing(3),
+    section: {
```
You can set most of these properties directly on the [`Grid` component](https://mui.com/api/grid/) directly.

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
     root: {
         width: '100%',
-        height: '100%',
+        minHeight: '85vh',
```
Hmm, what's the issue that's fixed by setting vertical height?

claynaut replied:
Running the page again, I'm not sure what the difference makes so I'll revert to the original setting. I think there may have been an issue with the footer overlapping the content, but I can't remember clearly.

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
 }));
 
-const AboutUs = () => {
+export default function AboutUs() {
```
Nitpicky comment: I've been trying to move the project away from using default exports.  Please remove `default`.

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
-                    eleifend.
+            <Grid item xs={12} sm={4} md={3} className={classes.section}>
+                <Image
+                    src='https://i.ibb.co/4YYwHx8/directors.png' 
```
We should try building this image directly into our assets rather than externally linking it.

DevelopmentByDavid commented on app/client/src/pages/aboutus.tsx:
```diff
@@ -1,52 +1,111 @@
-                    et facilisis urna. Praesent placerat lectus ac fermentum tincidunt. Nulla molestie suscipit lectus,
-                    eget maximus dolor tincidunt ut. Aenean sollicitudin varius laoreet. Fusce aliquet non nibh
-                    convallis venenatis.
+            <Grid item xs={12} className={classes.section}>
+                <Hidden mdUp>
+                    <Typography variant='body1' className={classes.paragraph}>
+                        In collaboration with the Congressional Management Foundation,  Prof. 
+                        Esterling and Prof. Salloum co-led the Connecting to Congress (C2C) 
+                        research team that received a grant from the National Science Foundation 
+                        (NSF) to work with the U.S. Congress on developing new technology for 
+                        constituent engagement. The C2C idea was that, in today’s society, it is 
+                        hard for individuals to feel like their voice is heard on important 
+                        policies.
+                    </Typography>
+                </Hidden>
+                <Typography variant='body1' className={classes.paragraph}>
+                    With seed funding from the Democracy Fund, the UCR TeCD-Lab developed 
+                    the <b>Prytaneum project</b>, the result of over a decade of research into 
+                    the best practices for online town halls. This new, open source online town hall 
+                    platform meets the requirements identified in the extensive C2C research, 
+                    and creates a setting for a many-participant, interactive event that lends 
+                    itself to collaborative, constructive exchanges between government officials 
+                    and constituents.
+                </Typography>
+                <Typography variant='body1' className={classes.paragraph}>
+                    The C2C research team published their findings  from the NSF study, Politics 
+                    with the People: Building a Directly Representative Democracy (Neblo, 
+                    Esterling, and Lazer 2018). In it they give recommendations for the design 
+                    features that make for a good online town hall experience. 
+                </Typography>
+                <Typography variant='body1' className={classes.paragraph}>
+                    In particular, the online platform should reduce barriers to participating 
+                    such that it is easy for everyone in the community to participate 
+                    interactively. The elected official should participate in the session 
+                    through streaming video and audio, so that the constituents can feel the 
+                    direct connection with the elected official and authenticate that it is in 
+                    fact the elected official with whom they are talking.
                 </Typography>
             </Grid>
         </Grid>
```
Hmm, do you mind elaborating why there's the same text multiple times?  I'm unclear on the reasoning.

claynaut replied:
There's the same text multiple times to be arranged differently based on the screen size. I'll try and find a better solution.

DevelopmentByDavid replied:
If you want it to be responsive typically we just use `Grid` component, but I see you're already doing that.  If you want to re-arrange based on the screen size, then you should make each piece of text a component and then arrange them based on the screen size.  This strategy allows for just repeating components rather than entire blocks of text.

DevelopmentByDavid reviewed (changes requested):
Meant to request changes in my previous review

kadey001 committed:
chore(env): updated to local vars

kadey001 committed:
feat(backend): added resolvers and methods for livefeedback

kadey001 committed:
feat(frontend): added live feedback ui, fragments, and mutations

kadey001 committed:
fix(live feedback): fixed live feedback resolver and subscription

kadey001 committed:
ci(gke): updated dev ingress to use dev.prytaneum.io

claynaut committed:
fix(frontend): implement requested changes to about us page

claynaut committed:
fix(frontend): implement requested changes to about us page

claynaut committed:
fix(frontend): implement requested changes to about us page

claynaut committed:
fix(frontend): implement requested changes to about us page

claynaut committed:
chore(frontend): remove unused imports

kadey001 committed:
ci(gke): updated managed cert adding dev subdomain

kadey001 committed:
fix(live feedback): fixed feedback order and moved list filtering

kadey001 committed:
Revert "ci(gke): updated dev ingress to use dev.prytaneum.io"

This reverts commit 28ec45d9fa1012df10b9519dd5948cd0c0858e08.

kadey001 committed:
Merge pull request #143 from kadey001/live-feedback

Live feedback feature

kadey001 committed:
Revert "Live feedback"

kadey001 committed:
Merge pull request #144 from c2c-project/revert-143-live-feedback

Revert "Live feedback"

kadey001 committed:
Revert "Revert "Live feedback""

kadey001 committed:
fix(update moderator): fixed build problem with unimplemented feature

kadey001 committed:
Merge pull request #146 from kadey001/hotfix

Build fix

kadey001 committed:
Merge pull request #145 from c2c-project/revert-144-revert-143-live-feedback

Revert the Revert, restoring the Live Feedback merge post hotfix.

kadey001 committed:
Fixed UpdateModerator type

kadey001 committed:
Fixed updateModerator

kadey001 committed:
Merge branch 'dev' of https://github.com/c2c-project/prytaneum into dev

kadey001 committed:
env fix

fixed db env file

kadey001 committed:
fix(db url env): fixed db env issue with deployment/development

kadey001 committed:
Merge branch 'dev' into db-env-deployment

kadey001 committed:
Merge pull request #147 from kadey001/db-env-deployment

fixed the db url so now it works when developing locally as well as providing the correct url when deploying via the dockerfile env var.

kadey001 committed:
Revert "Db env deployment"

kadey001 committed:
Merge pull request #148 from c2c-project/revert-147-db-env-deployment

Revert "Db env deployment"

kadey001 committed:
fix(env): updated database url env var to string w/o \

kadey001 committed:
Merge branch 'db-env-deployment' of https://github.com/kadey001/prytaneum into db-env-deployment

kadey001 committed:
Merge branch 'dev' into db-env-deployment

kadey001 committed:
Merge pull request #149 from kadey001/db-env-deployment

Db env deployment

kadey001 committed:
fix(feedback): updated to require being logged in to view live feedback

kadey001 committed:
build(next): updated next version

kadey001 committed:
fix(env): removed old unsued env vars

kadey001 committed:
build(typescript): updated typescript version to fix next errors

kadey001 committed:
fix(env): dev env fix

kadey001 committed:
chore(footer): temporarly removed footer until it is fixed

kadey001 committed:
Merge pull request #150 from kadey001/dev

Minor Fixes

kadey001 committed:
Update .env

Trying to fix production env issue

kadey001 committed:
Update .env

Temporarily reverting env changes to fix production build's connection to the database

kadey001 committed:
Update production-deployment.yml

Deploying to production by temporarily changing workflow since merging to master could cause issues currently

kadey001 committed:
Updated production-deployment.yml

Updated tag for docker images

kadey001 committed:
Update relay.ts

Updated wsProtocal to only wss to try and fix production issue.

DevelopmentByDavid commented:
Hmm, I think you need to merge c2c-project:dev into your about-us branch.  The diff is now 4k+ files since your branch is out of date.

claynaut commented:
Ah, ok. I'll try and get that fixed later.

claynaut committed:
chore(project): fix conflicts

claynaut commented:
> Hmm, I think you need to merge c2c-project:dev into your about-us branch. The diff is now 4k+ files since your branch is out of date.

Should be fixed now.

kadey001 committed:
Updated production-deployment.yml

Changed production deployment branch trigger back to master

kadey001 committed:
ci(deployment): trying to use pryatneum dev subdomain

kadey001 committed:
ci(dockerfile): added default env db vars and removed from env file

kadey001 committed:
ci(deployment): reverted dev deployment ingress

kadey001 committed:
ci(deployment): updaed managed cert name

kadey001 committed:
ci(certificate): added development certificate

kadey001 committed:
ci(certificate): fixed namespace for dev cert

kadey001 committed:
ci(deployment): updated database url env

kadey001 committed:
ci(env): updated db url to single secret var

kadey001 committed:
chore(packages): removed prytaneum typings

kadey001 committed:
fix(position): fixed position bug by increasing time slice used

kadey001 committed:
chore(dockerfile): cleaned up unessisary env vars

kadey001 committed:
Merge pull request #153 from kadey001/dev

Dev Deployment, Dev Environment Fix, and Bug Fix

kadey001 committed:
chore(redux): removing redux

kadey001 committed:
fix(ws): only use wss in prodution env

kadey001 committed:
fix(event sidebar): fixed sidebar tabs

kadey001 committed:
ci(deployment): updated to deploy on pull requests to dev

kadey001 committed:
ci(deployment): updated to deploy on pull requests to dev

kadey001 committed:
Merge branch 'unused-code-cleanup' of https://github.com/kadey001/prytaneum into unused-code-cleanup

kadey001 committed:
ci(deployment): removed on pull request

kadey001 committed:
Merge pull request #160 from kadey001/unused-code-cleanup

Event Sidebar Tabs Refactor & Redux Removal

kadey001 committed:
Update development-deployment.yml

Updated Grahpql url

kadey001 committed:
fix(card size bug): long words now break and wrap correctly within a question card

kadey001 committed:
chore(event sidebar): updated default tab

kadey001 committed:
fix(tabs): made tabs scrollable so all can be seen on mobile

kadey001 committed:
fix(user settings): cleaned up and fixed logged out issue on user settings page

kadey001 committed:
fix(logout): fixed user menu logout with useLogout hook

kadey001 committed:
fix(login): updated login onSuccess route

kadey001 committed:
fix(protected routes): protected routes depending on users login status

kadey001 committed:
chore(cleanup): code cleanup

kadey001 committed:
Merge branch 'dev' of https://github.com/kadey001/prytaneum into next-update

kadey001 committed:
ci(deployment): deploy on pull request target

kadey001 committed:
chore(next + lint): upgraded next version and added linting for the client

DevelopmentByDavid commented:
Hmm, it's still 4.5k file diff.  I think instead of a merge there needed to be a rebase.  Are you able to try that out?  This may be a good resource to read up on it https://www.atlassian.com/git/tutorials/merging-vs-rebasing  

It was probably confusing that I said to merge the changes in, apologies.

kadey001 committed:
chore(useLogout): code cleanup + relay compile fix

kadey001 committed:
fix(user context): refreshing on protected routes now works properly

kadey001 committed:
chore(eslint): updated file extension for consistancy

kadey001 committed:
chore(eslintrc): added next to extends field

kadey001 committed:
ci(linting): added linting to github actions workflow

kadey001 committed:
ci(linting): updated script

kadey001 committed:
ci(linting): updated steps

kadey001 committed:
ci(linting): testing max-warnings flag

kadey001 committed:
fix(linting): fixed some linter warnings

kadey001 committed:
chore(husky): added linting to husky pre-commit

kadey001 committed:
fix(next): downgraded next version slightly to fix yarn pnp issue

kadey001 committed:
fix(eslint): downgraded eslint to v7 to work with next version

kadey001 committed:
Merge pull request #163 from kadey001/next-update

Next + Typescript Upgrade & Client Linting

claynaut commented:
> Hmm, it's still 4.5k file diff. I think instead of a merge there needed to be a rebase. Are you able to try that out? This may be a good resource to read up on it https://www.atlassian.com/git/tutorials/merging-vs-rebasing
> 
> It was probably confusing that I said to merge the changes in, apologies.

Ah, ok. I'll try rebasing it.

kadey001 committed:
ci(env): added mailgun env vars to deployment

kadey001 committed:
feat(logger): added winston logger

kadey001 committed:
chore(logging): updated console logs to use logger

kadey001 committed:
fix(service worker): added unregistering for old service workers

kadey001 committed:
fix(eslint): fixed eslint error with service worker unregistering

kadey001 committed:
fix(sidenav): disabled movingbg component to address mobile ui problem

kadey001 committed:
Merge branch 'dev' into dev

kadey001 committed:
Merge pull request #161 from kadey001/dev

UI Bug Fixes + Service Worker Fix

kadey001 committed:
ci(workflow): updated deployment workflow to only deploy on a pull request merge

kadey001 committed:
Merge branch 'dev' of https://github.com/c2c-project/prytaneum into dev

kadey001 committed:
Merge pull request #168 from kadey001/dev

Workflow Fix

kadey001 committed:
Merge branch 'dev' into login/logout-bugfix

kadey001 committed:
Merge pull request #162 from kadey001/login/logout-bugfix

Login/logout Bug Fixes

kadey001 committed:
ci(workflow): change the trigger

updated on pull_request to on pull_request_target

kadey001 committed:
ci(workflow): updated merge linting triggers

kadey001 committed:
chore(cleanup): removed unused commented code

kadey001 committed:
Merge pull request #169 from kadey001/dev

Workflow Test PR

kadey001 committed:
ci(workflow): removed a trigger

This extra trigger was causing the workflow to be run 2x as a duplicate.

kadey001 committed:
chore(winston): updated prod and dev loggers

updated build functions and naming conventions for loggers

kadey001 committed:
chore(server logging): updated server logging to use winston

errors and their stacktrace should now be logged when they happen.

kadey001 committed:
Merge pull request #170 from kadey001/logging

Improved Logging

kadey001 committed:
ci(linting): added back trigger

only the one trigger did not seem to work so the trigger is being re-added

kadey001 committed:
chore(formatting): updated workflows to prettier format

kadey001 committed:
Merge pull request #171 from kadey001/dev

Workflow Test

kadey001 committed:
chore(logging): added gcloud winston transport

kadey001 committed:
chore(logger): added service context

kadey001 committed:
Merge pull request #172 from kadey001/logging

GKE Logging

kadey001 committed:
ci(workflow): dev workflow refactor

kadey001 committed:
ci(workflow): removed old workflow files

kadey001 committed:
Merge pull request #173 from kadey001/workflow-refactor

Workflow refactor

kadey001 committed:
chore(eslint): added eslint packages and script to server

kadey001 committed:
fix(lint): fixed linting error

kadey001 committed:
chore(eslint): updated script to have max warnings set as 10

kadey001 committed:
ci(workflow): added server linting step

kadey001 committed:
fix(linting): fixed a number of linting warnings

kadey001 committed:
Merge pull request #174 from kadey001/server-linting

Server linting

kadey001 committed:
ci(workflow): updated workflow

trying a different condition to see if deployment will trigger properly.

kadey001 committed:
ci(workflow): updated dev workflow

still attempting to have deployment run on pr merge

kadey001 committed:
Workflow Fix (#175)

updated workflow to try and get deployment triggering on merge

kadey001 committed:
Revert "Workflow Fix (#175)"

This reverts commit f20850a244172463cd345c181df866a787b7f40f.

kadey001 committed:
Workflow fix (#177)

* ci(workflow): updated workflow

kadey001 committed:
Revert "Workflow fix (#177)"

This reverts commit 7cf72cd37aea4e29031f42ec002f8c127d096b0c.

kadey001 committed:
Merge pull request #178 from c2c-project/revert-177-workflow-fix

Revert "Workflow fix"

kadey001 committed:
Workflow fix (#179)

* ci(workflow): updated workflow

kadey001 committed:
ci(workflow): removed a trigger

Removed the pull_request trigger since it fails on PRs from a forked repo

kadey001 committed:
fix(feedback card): text now wraps properly

kadey001 committed:
Merge pull request #180 from kadey001/live-feedback-bugfix

Feedback Card Bugfix

kadey001 committed:
ci(workflow): updated workflow name

shortened workflow name

kadey001 committed:
fix(logout): fixed main logout bugs

kadey001 committed:
style(useLogout): minor style changes and removing unused imports

kadey001 committed:
refactor(org profile): removed redundant hook

parent already checks for users auth status

kadey001 committed:
Merge pull request #182 from kadey001/#181-logout-bugfixes

#181 logout bugfixes

kadey001 committed:
chore(fastify): updated fastify version + added uuid package

kadey001 committed:
build(logging): updated to use fastify logger for all logging

updated data that is output with the logs making them more informative and useful

kadey001 committed:
build(winston): removed winston logger

winston logger has been replaced by the built in fastify pino logger

kadey001 committed:
Merge branch 'c2c-project:dev' into mailgun

kadey001 committed:
Merge branch 'c2c-project:dev' into server-logging

kadey001 committed:
ci(workflow): updated dev workflow triggers

Removed the on push trigger as workflow only really needs to run on the pull_request_target events. It is redundant to also trigger them on pushes

kadey001 committed:
Merge branch 'c2c-project:dev' into server-logging

kadey001 committed:
Merge branch 'c2c-project:dev' into mailgun

kadey001 committed:
refactor(mailgun): simplified sendEmail function to make it more generic and reuseable

kadey001 committed:
refactor(email invite): cleaned up commented code and updated email invite to use a template

kadey001 committed:
Merge pull request #183 from kadey001/server-logging

Improved Server Logging

kadey001 committed:
Merge branch 'c2c-project:dev' into mailgun

kadey001 committed:
Merge pull request #184 from kadey001/mailgun

Mailgun Configuring/Refactor

kadey001 committed:
ci(workflow): removed duplicate event trigger

claynaut committed:
chore(project): rebase and fix conflicts
